src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -161,6 +161,45 @@ export default async function UserBreaches() {
     return breachRowsHTML;
   }
 
+  // Fetch list of subscriptions.
+  const bearerToken = session?.user.subscriber?.fxa_access_token;
+  if (!bearerToken) {
+    console.error("User has no bearer token");
+  }
+  const result = await fetch(
+    `${process.env.OAUTH_API_URI}/oauth/mozilla-subscriptions/customer/billing-and-subscriptions`,
+    {
+      headers: {
+        "Content-Type": "application/json",
+        authorization: `Bearer ${bearerToken}`,
+      },
+    }
+  );
+  let subscriptions;
+  if (result.ok) {
+    subscriptions = await result.json();
+  } else {
+    console.error("Could not refresh subscriptions:", result);
+  }
+
+  function userSubscribed(subscriptions: any) {
```

Comment:
Nit, but I often find it useful if functions start with a verb, and functions that return a boolean to start with `is`, `has`, `contains`, or similar. So this could be e.g. `isUserSubscriber` or `includesPremiumSubscription`.